contracts/exchange-libs/contracts/src/MixinLibMathRichErrors.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.5.9;
+
+import "@0x/contracts-utils/contracts/src/RichErrors.sol";
+import "./interfaces/IMixinLibMathRichErrors.sol";
+
+
+contract MixinLibMathRichErrors is
+    IMixinLibMathRichErrors,
+    RichErrors
+{
+    // solhint-disable func-name-mixedcase
+    function DivisionByZeroError()
+        internal
+        pure
+        returns (bytes memory)
+    {
+        bytes4 divisionError = DIVISION_BY_ZERO_SELECTOR;
+        assembly {
```

Comment:
This works, but you could go a step further with `return "\xa7\x91\x83\x7c";` :) Maybe just store this value as a constant `DIVISION_BY_ZERO_ERROR` in `IMixinLibMathRichErrors.sol`?